libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/hwdef.dat

```diff
@@ -0,0 +1,160 @@
+
+# hw definition file for processing by chibios_hwdef.py
+# for IFLIGHT_2RAW_H743 hardware.
+# thanks to betaflight for pin information
+
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# board ID for firmware load
+APJ_BOARD_ID AP_HW_2RAWH743
+
+# crystal frequency, setup to use external oscillator
+OSCILLATOR_HZ 8000000
+
+MCU_CLOCKRATE_MHZ 480
+
+FLASH_SIZE_KB 2048
+
+# bootloader takes first sector
+FLASH_RESERVE_START_KB 384
+
+define HAL_STORAGE_SIZE 16384
+define STORAGE_FLASH_PAGE 1
+
+STM32_ST_USE_TIMER 2
+
+# SPI devices
+
+# SPI1
+PA5 SPI1_SCK SPI1
+PA6 SPI1_MISO SPI1
+PD7 SPI1_MOSI SPI1
+
+# SPI2
+PB13 SPI2_SCK SPI2
+PB14 SPI2_MISO SPI2
+PB15 SPI2_MOSI SPI2
+
+# Chip select pins
+PA15 SDCARD1_CS CS
+PB12 OSD1_CS CS
+PC15 GYRO1_CS CS
+
+# SERIAL ports
+SERIAL_ORDER OTG1 USART1 USART2 USART3 UART4 EMPTY USART6 OTG2
+# PA10 IO-debug-console
+PA11 OTG_FS_DM OTG1
+PA12 OTG_FS_DP OTG1
+
+# USART1 (VTX HD)
+PA10 USART1_RX USART1
+PA9 USART1_TX USART1
+# The board doesn't boot if this is set
+#define DEFAULT_SERIAL1_PROTOCOL SerialProtocol_MSP_DisplayPort
+define DEFAULT_SERIAL1_PROTOCOL SerialProtocol_None
```

Comment:
this needs to be set to DisplayPort by default

Reply:
Fixed